widget-reference/table.md

```diff
@@ -63,3 +63,4 @@ Tables are useful to view large lists of data. To drill down into the data of a
 | **onRowSelected** | Sets the action to be run when user selects a row in the table. Default supported actions are: Call API, Execute DB Query, Navigate to Page, Show Alert, Open / Close popup |
 | **onPageChange** | Sets the action to be run when a table page is changed. Default supported actions are: Call API, Execute DB Query, Navigate to Page, Show Alert, Open / Close popup |
```

Comment:
@gogetter22 any idea why the table is transparent in your gif instead of white?

Reply:
@Nikhil-Nandagopal : I looked at it again and it's white. Do you mean to say that table is see-through i.e. background is seen or something?

Reply:
@gogetter22 yes it doesn't look like the table here
https://docs.appsmith.com/core-concepts/building-the-ui

Reply:
@Nikhil-Nandagopal  - Got the issue. While capturing the video in MP4, coloring was fine but an online tool changed its color. Converted to GIF with another tool, please review now.